app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,37 @@
+class RelationshipsController < ApplicationController
+  before_action :require_login
+
+  def create
+    @user_to_follow = params[:relationship][:followed_id].to_i
+
+    if params[:relationship][:followed_id] != current_user.id
+      current_user.follow(@user_to_follow)
+      redirect_to profile_path(@user_to_follow)
+    else
+      flash[:notice] = 'error'
```

Comment:
Give the user descriptive messages as to what went wrong. In this case, "Unable to create relationship" would work.